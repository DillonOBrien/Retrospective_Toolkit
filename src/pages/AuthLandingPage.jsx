import React from 'react';
import '../styles/LandingPage.css';
import bg from '../resources/fin.png';


function AuthLandingPage() {

    return (
        <div class="centered">
            <img src={bg} 
                alt="fin"
                height={318}
                width={546} 
            />
        </div>
    );
}

export default AuthLandingPage;