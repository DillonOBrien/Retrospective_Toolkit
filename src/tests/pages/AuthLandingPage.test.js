import React from "react";
import { shallow } from "enzyme";
import AuthLandingPage from "../../pages/AuthLandingPage";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AuthLandingPage />);
});

describe("Pages/AuthLandingPage Testing", () => {
  // Snapshot for AuthLandingPage
  test("SNAPSHOT: AuthLandingPage", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
