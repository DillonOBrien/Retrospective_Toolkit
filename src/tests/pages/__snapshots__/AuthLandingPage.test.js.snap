// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/AuthLandingPage Testing SNAPSHOT: AuthLandingPage 1`] = `
<div>
  <div
    className="centered"
  >
    <img
      alt="fin"
      height={318}
      src="https://d1g31diwtzkeb3.cloudfront.net/fin.png"
      width={546}
    />
  </div>
</div>
`;
